src/content_scripts/common.js

```diff
@@ -3,14 +3,13 @@
 /* eslint-disable no-unused-vars */
 /* eslint-disable prefer-const */
 
+/* globals browser */
```

Comment:
Usually our EsLint config file includes this already, i.e. it knows we are operating as a web extension.
As such, you should be able to remove this here.